bundles/org.openhab.binding.darksky/src/main/resources/ESH-INF/config/config.xml

```diff
@@ -24,31 +24,37 @@
 				<option value="az">Azerbaijani</option>
 				<option value="be">Belarusian</option>
 				<option value="bg">Bulgarian</option>
+				<option value="bn">Bengali</option>
 				<option value="bs">Bosnian</option>
 				<option value="ca">Catalan</option>
 				<option value="cs">Czech</option>
 				<option value="da">Danish</option>
 				<option value="de">German</option>
 				<option value="el">Greek</option>
 				<option value="en">English</option>
+				<option value="eo">Esperanto</option>
 				<option value="es">Spanish</option>
 				<option value="et">Estonian</option>
 				<option value="fi">Finnish</option>
 				<option value="fr">French</option>
 				<option value="he">Hebrew</option>
+				<option value="hi">Hindi</option>
 				<option value="hr">Croatian</option>
 				<option value="hu">Hungarian</option>
 				<option value="id">Indonesian</option>
 				<option value="is">Icelandic</option>
 				<option value="it">Italian</option>
 				<option value="ja">Japanese</option>
 				<option value="ka">Georgian</option>
+				<option value="kn">Kannada</option>
```

Comment:
As a side note: I did not change spellings - just copy & pasted them from https://status.darksky.net/2019/02/13/model-updates.html.